internal/cli/apps.go

```diff
@@ -62,6 +63,43 @@ var CmdAppList = &cobra.Command{
 	},
 }
 
+// CmdAppCreate implements the epinio `apps create` command
+var CmdAppCreate = &cobra.Command{
+	Use:   "create NAME",
+	Short: "Create just the app, without creating a workload",
+	Args:  cobra.ExactArgs(1),
+	RunE: func(cmd *cobra.Command, args []string) error {
+		client, err := clients.NewEpinioClient(cmd.Context(), cmd.Flags())
+
+		if err != nil {
+			return errors.Wrap(err, "error initializing cli")
+		}
+
+		err = client.AppCreate(args[0])
+		if err != nil {
+			return errors.Wrap(err, "error creating app")
+		}
+
+		return nil
+	},
+	SilenceErrors: true,
+	SilenceUsage:  true,
```

Comment:
This should be adapted to the changes in the other commands, i.e. `Silence*: false` by default, and `SilenceUsage` dynamically activated in the command function. See PR #539 . This is something which can be done after this is merged. I.e. a nit, and I do not wish to cause more delay.